src/server/hll_family_test.cc

```diff
@@ -194,6 +194,8 @@ TEST_F(HllFamilyTest, MergeOverlapping) {
 }
 
 TEST_F(HllFamilyTest, MergeInvalid) {
+  GTEST_SKIP() << "TBD: MergeInvalid test fails with multi-shard runs, see #5004";
```

Comment:
but we got experimental flag off, no need to disable this. Also another test case in multi failed for multi shard. I will open an issue about. Thank you for opening the other one